### ISSUE TITLE
GetCellValue does not return raw value, even with the option RawCellValue: true

### ISSUE THREAD

**Issue by ffleader1** [qax-os/excelize#1161]
Unless I am reading the doc from, I suppose by passing the GetCellValue with the option RawCellValue set to True:

````
	cell, err := f.GetCellValue(sheet, cellLoc, excelize.Options{
		RawCellValue: true,
	})
````

I can get the raw text of what has been written in the cell.
Does not seem to be the case though.
The above line would gimme me the result of the formula, a number, but I need is the formula string, like ```=SUM(O4,N4)```
So, is this a bug, or am I not looking correctly?
Thank you

**Comment by xuri** [comment]
Thanks for your issue. Please use [`SetCellFormula`](https://pkg.go.dev/github.com/xuri/excelize/v2#File.GetCellFormula) to get the formula from the cell and use [`GetCellType`](https://pkg.go.dev/github.com/xuri/excelize/v2#File.GetCellType) to determine which cell is a formula cell.

**Comment by ffleader1** [comment]
> 

You mean to use ```GetCellFormula``` right?
Well, the thing is if the Cell contains ```=SUM(O4,N4)```, then the formula will be ```O4+N4```.
I need the formula string due to the fact that that my Cell has ```[LET](https://support.microsoft.com/en-us/office/let-function-34842dd8-b92b-4d3f-b325-b8b8f9908999)``` function, which cannot be parsed (yet), so I need the raw string formular to run it through a custom LET parser that I wrote.
But I cannot because if the formula has LET, it will not work properly. But I cannot get the ră string formular.
Basically, I just want to know how (or is it possible) to get the Raw string fomular.
Thank you.

**Comment by ffleader1** [comment]
> Thanks for your issue. Please use [`SetCellFormula`](https://pkg.go.dev/github.com/xuri/excelize/v2#File.GetCellFormula) to get the formula from the cell and use [`GetCellType`](https://pkg.go.dev/github.com/xuri/excelize/v2#File.GetCellType) to determine which cell is a formula cell.

> 

You mean to use ```GetCellFormula``` right?
Well, the thing is if the Cell contains ```=SUM(O4,N4)```, then the formula will be ```O4+N4```.
I need the formula string due to the fact that that my Cell has ```[LET](https://support.microsoft.com/en-us/office/let-function-34842dd8-b92b-4d3f-b325-b8b8f9908999)``` function, which cannot be parsed (yet), so I need the raw string formular to run it through a custom LET parser that I wrote.
But I cannot because if the formula has LET, it will not work properly. But I cannot get the raw string formular.
Basically, I just want to know how (or is it possible) to get the Raw string fomula.
Thank you.

**Comment by ffleader1** [comment]
Any update on this?
I just want to get the raw raw formular, not the formula that has been passed. Thank you.

**Comment by xuri** [comment]
Hi @pgalbavy-itrs, thanks for your feedback, could you provide your input file attachment without confidential info?

**Comment by pgalbavy-itrs** [comment]
Will try - it turned out the cell was formatted by the user with some weird combination of languages and numeric types. I will have to go find a copy of the XLSX file before they fixed it on our advice.
